static/app/views/profiling/profileDetails/components/profileDetailsTable.tsx

```diff
@@ -279,6 +279,25 @@ function ProfilingFunctionsTableCell({
         </Container>
       );
     }
+    case 'symbol': {
+      return (
+        <Container>
+          <Link
+            to={generateProfileFlamechartRouteWithQuery({
+              orgSlug: orgId,
+              projectSlug: projectId,
+              profileId: eventId,
+              query: {
+                highlightAll: delimiter.join(dataRow.symbol, dataRow.image),
+                tid: (dataRow.thread ?? dataRow.tids?.[0]) as string,
```

Comment:
right now we'll navigate to the first `tid`; i would rather the user have a selection, but i will add that in a follow up PR.